packages/server/src/semester/setCourse.command.ts

```diff
@@ -0,0 +1,100 @@
+import { Command, Positional } from 'nestjs-command';
+import { Injectable } from '@nestjs/common';
+import { Season } from '@koh/common';
+import { SemesterService } from './semester.service';
+import { SemesterModel } from './semester.entity';
+
+@Injectable()
+export class SetSemesterCommand {
+  constructor(private readonly semService: SemesterService) {}
+  @Command({
+    command: 'semester:modsemester <mode> <semester> <year>',
+    describe: '',
+    autoExit: true,
+  })
+  async create(
+    @Positional({
+      name: 'mode',
+      describe: 'mode: either enable or disable.',
+      type: 'string',
+    })
+    mode: string,
+    @Positional({
+      name: 'semester',
+      describe: 'the semester to enable',
+      type: 'string',
+    })
+    semester: string,
+    @Positional({
+      name: 'year',
+      describe: 'year to enable',
+      type: 'number',
+    })
+    year: number,
+  ): Promise<void> {
+    if (Number.isNaN(year)) {
+      console.log('Invalid year: Please provide the year as a number.');
+      return;
+    }
+
+    const sem = this.validateSemester(semester);
+    if (!sem) {
+      console.log('Invalid season: "' + semester + '"');
+      console.log(
+        'Pick from one of:' +
+          '\n-  "Fall"' +
+          '\n-  "Spring"' +
+          '\n-  "Summer_1"' +
+          '\n-  "Summer_2"' +
+          '\n-  "Summer_Full"',
+      );
+      return;
+    }
+
+    let enOrDisable = mode === 'enable';
```

Comment:
I think since you have the switch statement to correctly assign enOrDisable, there is no need to set it to `mode === 'enable'`. You can just declare the variable using let and assign it below